rust/types/src/markets.rs

```diff
@@ -60,27 +65,33 @@ pub struct TickerUpdate {
     pub timestamp: u64,
 }
 
+/// A market is where two assets are exchanged. Most notably, in a `BTC/USDC` pair
+/// `BTC` is the base and `USDC` is the quote.
 #[derive(Debug, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct Market {
+    /// Identifier
     pub symbol: String,
+    /// Left-hand side asset
```

Comment:
Are the base and quote assets the more commonly used terms for a market?
```suggestion
    /// The base asset.
```

Reply:
Yes, they usually are. LHS and RHS were used to avoid duplicated words with the struct-level documentation.